pkg/users/accounts-list.js

```diff
@@ -183,17 +250,82 @@
     });
 };
 
-const AccountsList = ({ accountsInfo, current_user, groups }) => {
-    const [currentTextFilter, setCurrentTextFilter] = useState("");
+const GroupsList = ({ groups, accounts }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+    const columns = [
+        { title: _("Group name"), sortable: true },
+        { title: _("ID"), sortable: true },
+        { title: _("# of users"), sortable: true },
+        { title: _("Accounts") },
+    ];
 
-    const accounts = mapGroupsToAccount(accountsInfo, groups);
+    const sortRows = (rows, direction, idx) => {
+        // GID and members columns are numeric
+        const isNumeric = idx == 1 || idx == 2;
+        const sortedRows = rows.sort((a, b) => {
+            const aitem = a.columns[idx].sortKey || a.columns[idx].title;
+            const bitem = b.columns[idx].sortKey || b.columns[idx].title;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17980-20221206-121249-7589670f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)